spk/mosh/Makefile

```diff
@@ -0,0 +1,27 @@
+SPK_NAME = mosh
+SPK_VERS = 1.3.2
+SPK_REV = 1
+SPK_ICON = src/mosh.png
+DSM_UI_DIR = app
+
+DEPENDS = cross/$(SPK_NAME)
+
+MAINTAINER = fredrike
+DESCRIPTION = Mosh is a remote terminal application that supports intermittent connectivity, allows roaming, and provides speculative local echo and line editing of user keystrokes.
+RELOAD_UI = no
+DISPLAY_NAME = Mosh
+STARTABLE = no
+CHANGELOG = """
+
+HOMEPAGE = http://mosh.org
+LICENSE  = GPLv3+
+
+REQUIRED_DSM = 5.0
+
+INSTALLER_SCRIPT = src/installer.sh
+SSS_SCRIPT       = src/dsm-control.sh
```

Comment:
I would say this script is useless. Please try removing script file and reference

Reply:
Many other packages use `dsm-control.sh` so I figured it is mandatory.